index.bs

```diff
@@ -325,8 +319,20 @@ interface PictureInPictureWindow {
 };
 </pre>
 
-The {{PictureInPictureWindow}} is a simple proxy to get information such as
-dimensions about the current Picture-in-Picture window.
+A {{PictureInPictureWindow}} instance represents a <a>Picture-in-Picture
+window</a> associated with an {{Element}}. When instantiated, an instance of
+{{PictureInPictureWindow}} has its |state| set to |opened|.
+
+When the <dfn>close window algorithm</dfn> with an instance of
+{{PictureInPictureWindow}} is invoked, its |state| is set to |closed|.
+
+The {{width}} attribute MUST return the width in <a lt=px value>CSS pixels</a> of the
+<a>Picture-in-Picture window</a> associated with {{pictureInPictureElement}} if
+the |state| is |opened|. Otherwise, it MUST return 0.
+
+The {{height}} attribute is the height in <a lt=px value>CSS pixels</a> of the
```

Comment:
The wording here is different from above. Can you update it?

Reply:
Oops. Thanks!